core/src/test/java/com/graphhopper/routing/util/EncodingManagerTest.java

```diff
@@ -66,6 +71,13 @@ public void testEncoderAcceptNoException()
     }
 
     @Test
+    public void testEncoderWithWrongVersionIsRejected()
+    {
+        thrown.expect(IllegalArgumentException.class);
```

Comment:
Yes, there is no way to test exception message using annotation, but i think it's doesn't matter in tests like this :).